Makefile.js

```diff
@@ -531,8 +531,8 @@ target.lintDocsJS = function([fix = false] = []) {
 };
 
 target.fuzz = function({ amount = 1000, fuzzBrokenAutofixes = false } = {}) {
-    const fuzzerRunner = require("./tools/fuzzer-runner");
-    const fuzzResults = fuzzerRunner.run({ amount, fuzzBrokenAutofixes });
+    const { run } = require("./tools/fuzzer-runner");
+    const fuzzResults = run({ amount, fuzzBrokenAutofixes });
```

Comment:
[Question] For this and the other changes outside `knip.jsonc`, could you explain them please? At first glance it feels like a bug in Knip that any non-functional changes like this would need to be made.

Reply:
Please see https://github.com/eslint/eslint/pull/18005#discussion_r1458414116